File: sample/src/main/java/ru/terrakok/cicerone/sample/mvp/main/SampleView.java
package ru.terrakok.cicerone.sample.mvp.main;


import moxy.MvpView;
import moxy.viewstate.strategy.AddToEndSingleStrategy;
import moxy.viewstate.strategy.StateStrategyType;

/**
 * Created by Konstantin Tckhovrebov (aka @terrakok)
 * on 11.10.16
 */

@StateStrategyType(AddToEndSingleStrategy.class)
public interface SampleView extends MvpView {
    void setTitle(String title);
}

File: sample/src/main/java/ru/terrakok/cicerone/sample/mvp/main/SampleView.kt
package ru.terrakok.cicerone.sample.mvp.main

import com.arellomobile.mvp.MvpView
import com.arellomobile.mvp.viewstate.strategy.AddToEndSingleStrategy
import com.arellomobile.mvp.viewstate.strategy.StateStrategyType

/**
 * Created by Konstantin Tckhovrebov (aka @terrakok)
 * on 11.10.16
 */
@StateStrategyType(AddToEndSingleStrategy::class)
interface SampleView : MvpView {
    fun setTitle(title: String)
}